code/defines/procs/statistics.dm

```diff
@@ -70,25 +69,23 @@ proc/sql_report_death(var/mob/living/carbon/human/H)
 	if(!dbcon.IsConnected())
 		log_game("SQL ERROR during death reporting. Failed to connect.")
 	else
-		var/DBQuery/query = dbcon.NewQuery("INSERT INTO death (name, byondkey, job, special, pod, tod, laname, lakey, gender, bruteloss, fireloss, brainloss, oxyloss, coord) VALUES ('[sqlname]', '[sqlkey]', '[sqljob]', '[sqlspecial]', '[sqlpod]', '[sqltime]', '[laname]', '[lakey]', '[H.gender]', [H.getBruteLoss()], [H.getFireLoss()], [H.brainloss], [H.getOxyLoss()], '[coord]')")
+		var/DBQuery/query = dbcon.NewQuery("INSERT INTO [format_table_name("death")] (name, byondkey, job, special, pod, tod, laname, lakey, gender, bruteloss, fireloss, brainloss, oxyloss, coord) VALUES ('[sqlname]', '[sqlkey]', '[sqljob]', '[sqlspecial]', '[sqlpod]', '[sqltime]', '[laname]', '[lakey]', '[H.gender]', [H.getBruteLoss()], [H.getFireLoss()], [H.brainloss], [H.getOxyLoss()], '[coord]')")
 		if(!query.Execute())
 			var/err = query.ErrorMsg()
 			log_game("SQL ERROR during death reporting. Error : \[[err]\]\n")
 
 
-proc/sql_report_cyborg_death(var/mob/living/silicon/robot/H)
-	if(!sqllogging)
+/proc/sql_report_cyborg_death(mob/living/silicon/robot/H)
+	if(!config.sql_enabled)
 		return
 	if(!H)
 		return
 	if(!H.key || !H.mind)
 		return
 
-	var/turf/T = get_turf(H)
-	var/area/placeofdeath = get_area(T)
-	var/podname = sanitizeSQL("Unknown Area")
-	if(placeofdeath)
-		podname = sanitizeSQL(placeofdeath.name)
+	var/turf/T = H.loc
+	var/area/placeofdeath = get_area(T.loc)
+	var/podname = placeofdeath.name

```

Comment:
this probably should `sanitizeSQL()` still; Players can create and rename areas.